packages/ra-data-simple-rest/package.json

```diff
@@ -19,8 +19,8 @@
     "bugs": "https://github.com/marmelab/react-admin/issues",
     "license": "MIT",
     "scripts": {
-        "build": "tsup src/index.ts --silent --clean --format cjs,esm --minify --keep-names --metafile --sourcemap --dts --legacy-output",
-        "watch": "tsup src/index.ts --silent --clean --format cjs,esm --minify --keep-names --metafile --sourcemap --dts --legacy-output --watch"
+        "build": "tsup --external react-hook-form src/index.ts --silent --clean --format cjs,esm --minify --keep-names --metafile --sourcemap --dts --legacy-output",
```

Comment:
do we need to add it even in packages that don't have a dependency on react-hook-form?

Reply:
I'd rather be safe than sorry

Reply:
Removed from packages where it was unnecessary